static/js/util/util.js

```diff
@@ -455,3 +455,30 @@ export function getUserDisplayName(profile: Profile): string {
 
   return `${first} ${last}${preferred_name}`;
 }
+
+function intersperse(arr, elementFunc) {
+  let resultArray = [], i=0;
+  if (i < arr.length)
+    resultArray.push(arr[i++]);
+  while (i < arr.length)
+    resultArray.push(elementFunc(i), arr[i++]);
+  return resultArray;
+}
```

Comment:
I think you might be able to use `R.chain` productively here - something like:

```js
const intersperse = R.curry((cb, xs) => {
  let inter = x => [x, cb(x)];
  return R.chain(inter, xs);
});
```

or in a more point-free form:

```js
const intersperse = cb => {
  let inter = x => [x, cb(x)];
  return R.chain(inter);
};
```